cif_pow.dic

```diff
@@ -11556,6 +11556,57 @@ save_PD_QPA_EXTERNAL_STD
     _name.object_id               PD_QPA_EXTERNAL_STD
     _category_key.name            '_pd_qpa_external_std.diffractogram_id'
 
+    _description_example.case
+;
+         data_ext_std_diffractogram
+            _pd_diffractogram.id         THE_REFERENCE
+            _pd_diffractogram.instr_id   labmachine
```

Comment:
This will need to be changed if my suggestion as to `instr_id` -> `detector_id` is accepted.